jobs/pulse3d/pulse3d-worker/src/lib/queries.py

```diff
@@ -0,0 +1,25 @@
+INSERT_INT0_UPLOADED_S3_OBJECTS = """
+    INSERT INTO uploaded_s3_objects(bucket, object_key, upload_started_at, uploading_computer_name)
+    VALUES ($1, $2, NOW(), $3);
+    """
+
+SELECT_LAST_UPLOAD_ID = """(SELECT id FROM uploaded_s3_objects ORDER BY id DESC LIMIT 1)"""
+
+INSERT_INTO_MANTARRAY_RECORDING_SESSIONS = """
+    INSERT INTO mantarray_recording_sessions(mantarray_recording_session_id, customer_account_id, user_account_id, instrument_serial_number, session_log_id, acquisition_started_at, length_microseconds,
+    recording_started_at)
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8);
+    """
+
+INSERT_INTO_MANTARRAY_RAW_FILES = f"""
+    INSERT INTO mantarray_raw_files(well_index, upload_id, length_microseconds, recording_started_at, mantarray_recording_session_id)
+    VALUES ($1, {SELECT_LAST_UPLOAD_ID}, $2, $3, $4);
```

Comment:
we might want to instead use a transaction and only select the last upload id once for all the inserts and then pass it as a normal parameter to the insert queries, depending on how this is used it might cause an issue doing multiple selects to get the last upload